allennlp/modules/seq2vec_encoders/__init__.py

```diff
@@ -0,0 +1,52 @@
+from typing import Type
+
+import torch
+
+from allennlp.common import Params
+from allennlp.common.checks import ConfigurationError
+from allennlp.experiments import Registry
+from allennlp.modules.seq2vec_encoders.wrapped_pytorch_rnn import WrappedPytorchRnn
+
+class _Wrapper:
+    """
+    The :class:`Registry` needs to have a ``Type[Seq2VecEncoder]`` as the value registered for each
+    key.  What that means is that we need to be able to ``__call__`` these values (as is done with
+    ``__init__`` on the class), and be able to call ``from_params()`` on the value.
+
+    In order to accomplish this, we have two options: (1) we create a ``Seq2VecEncoder`` class for
```

Comment:
Because I had a hard time thinking of how to even make it work.  Maybe I'm missing something, but there seemed to be problems with all of the ways I thought of doing this simply with the first option.

Because the rest of the code isn't affected by this decision, and this works, I'd vote for putting in a TODO to revisit this decision - the only lines that have to change are the lines in this module (and corresponding tests), and there aren't very many of them.